### DISCUSSION TITLE
Observability Of Kubernetes NO Data

### DISCUSSION THREAD

**Discussion by liyongxian** [apache/skywalking#10434] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Showcase (apache/skywalking-showcase)

### What happened

Skywalking Shows K8s Metrics, Deployed by https://github.com/apache/skywalking-showcase/tree/main/deploy/platform/kubernetes/feature-kubernetes-monitor,But NO Data
![image](https://user-images.githubusercontent.com/18571576/220816272-19f4bd90-5ae1-418a-b163-6a79b212efdf.png)

### What you expected to happen

Skywalking UI Can Show K8s Metrics.

### How to reproduce

Deployed by https://github.com/apache/skywalking-showcase/tree/main/deploy/platform/kubernetes/feature-kubernetes-monitor

### Anything else

kywalking Version: 9.3.0
OTel Version: opentelemetry-collector:0.71.0
Kubernetes Version:
![image](https://user-images.githubusercontent.com/18571576/220816453-da25a9e6-1da3-4a51-a6de-3d75c74cec62.png)

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
The demo.skywalking.apache.org is using this to monitoring AWS GKE. It is better you could check whether metrics have been forward to OAP. Your given information is not enough to tell anything.

**Reply by cybshark** [reply]
same isssue for me. I deploy Otlp collector and aget still not able to get data on dashboard

**Reply by lxx95** [reply]
Same problem with customized Skywalking cluster via HELM deploy (no skywalking-showcase) .
With v9.4.0 no problems, we have metrics in UI:
![image](https://github.com/apache/skywalking/assets/57501699/1851341c-7438-43be-8be0-898a5820e7cc)

Same configuration in **v9.5.0** show **NO DATA**.

Same for **VM metrics**.

I saw that the ENV variable related to OTEL METRICS was changed from v9.4.0 (**SW_OTEL_RECEIVER_ENABLED_OTEL_RULES**:"istio-controlplane") to v9.5.0 (**SW_OTEL_RECEIVER_ENABLED_OTEL_METRICS_RULES**:"istio-controlplane").

In my case the **variable was set according to the documentation page (and version)**.

**Reply by kezhenxu94** [reply]
Hi @lxx95 , sorry it turns out to be a bug, I modified the env var name in doc but didn't modify accordingly in the `yaml` configuration file. https://github.com/apache/skywalking/pull/11020 should fix it
